packages/framer-motion/src/components/AnimatePresence/types.ts

```diff
@@ -51,10 +51,18 @@ export interface AnimatePresenceProps {
      * )
      * ```
      *
-     * @beta
+     * @public
      */
     exitBeforeEnter?: boolean
 
+    /**
+     * If set to `true`, top-level `exit` components will be "popped" out of the DOM layout,
+     * so surrounding elements can animate to their new layout immediately.
+     *
+     * @public
+     */
+    popLayout?: boolean
```

Comment:
How does this interact with `exitBeforeEnter` and `presenceAffectsLayout`?
Just trying a bit:
- `popOnExit`
- `exitImmediately`
- `removeOnExit`
- `exitPops`

Reply:
Is there maybe a way to combine into `exitBehaviour: "pop" | "beforeEnter"`?

Reply:
`appearanceBehaviour: "popExit" | "deferredEnter"`

Reply:
~`method`~
~`action`~
`mode: "sync" | "popLayout" | "deferEnter"`
~`switch: "default" | "popLayoutExit" | "deferEnter"`~
~`swap`~

Reply:
Default is `sync`